server/index.js

```diff
@@ -6,7 +6,7 @@ app.use(express.static('public'));
 
 
 app.get('/api/imgt/:movieId', (req, res) => {
```

Comment:
The name of the API route is a little misleading, I'm not sure why you have two of these routes.

Reply:
That makes sense!  I'll adjust the route names so it is more clear - thank you!